src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/TraceConfigurationHelper.cs

```diff
@@ -0,0 +1,55 @@
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
+{
+    internal static class TraceConfigurationHelper
+    {
+        public static TracerProviderBuilder UseEnvironmentVariables(this TracerProviderBuilder builder, Settings settings)
+        {
+            return builder.SetResourceBuilder(ResourceBuilder.CreateDefault()
+                                                             .AddService(settings.ServiceName, serviceVersion: settings.ServiceVersion))
+                          .SetExporter(settings);
+        }
+
+        public static TracerProviderBuilder AddAspNetInstrumentation(this TracerProviderBuilder builder)
```

Comment:
`AddAspNetInstrumentation` will cause circular reference with this extension and OTel.SDK ASP.NET extension in line 18.

Reply:
How you found a solution for that?

Reply:
I am not able to test for framework at the moment :(

Reply:
Here the implementation that I got to work on Windows:
```csharp
        public static TracerProviderBuilder AddSdkAspNetInstrumentation(this TracerProviderBuilder builder)
        {
#if NET452 || NET461
            return builder.AddAspNetInstrumentation();
#elif NETCOREAPP3_1_OR_GREATER
            return builder.AddAspNetCoreInstrumentation();
#else
            return builder;
#endif
        }
```

Reply:
Took your code @pjanotti :)